ide/static/js/setParams.js

```diff
@@ -22,10 +22,19 @@ class SetParams extends React.Component {
     const net = this.props.net;
     let layer = net[this.props.selectedLayer];
     layer = JSON.parse(JSON.stringify(layer));
+    var intParams = ["crop_size", "num_output", "new_height", "new_width", "height", "width", "kernel_h", "kernel_w",
```

Comment:
Do we need to parse these as `int` at two places?

Reply:
Yeah because when layer is added it doesn't use setparams